src/CoreBundle/private/modules/MTTableEditor/MTTableEditor.class.php

```diff
@@ -217,6 +202,28 @@ public function Init()
         $this->AddURLHistory();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function checkAccessRightsOnTable(): bool
```

Comment:
I would avoid implementation details in the method name. For the call, it does not matter HOW or on WHAT the module checks access permission.

Reply:
That is then sub-issue to the new interface name.

What do you think of that? CheckTableAccessRightsInterface

Reply:
why table? why use AccessCheckInterface::checkAccess() (or assertAccess). 
That way we can just use one interface (AccessCheckInterface) instead of two (BackendPageAccessCheckInterface and CheckTableAccessRightsInterface).

Reply:
Hm, CheckTableAccessRightsInterface is still a solution for an old legacy case - and requires an "instanceof".

Whereas BackendPageAccessCheckInterface::checkPageAccess ist the more modern approach of rights on a page..